setup.py

```diff
@@ -1,14 +1,36 @@
-from setuptools import setup
+import glob
+import os
+import sys
 
-setup(
-    name='EPLaunch3',
-    version='0.1',
-    packages=['eplaunch', 'eplaunch.interface'],
-    url='https://energyplus.net',
-    license='ModifiedBSD',
-    author='DOE',
-    author_email='webmaster@doe.gov',
+from cx_Freeze import setup as cx_setup, Executable
+
+from eplaunch import NAME, VERSION
+
+include_files = []
+default_workflows = glob.glob("eplaunch/workflows/default/*.py")
+workflow_tuples = [(x, os.path.join('lib', x)) for x in default_workflows]
+include_files.extend(workflow_tuples)
```

Comment:
We have to include the "built-in" workflow files because they aren't automatically detected as a dependency on the project.